pkg/nuclide-outline-view/lib/OutlineView.js

```diff
@@ -173,7 +173,11 @@ class OutlineTree extends React.PureComponent {
       }
     };
 
-    const classes = classnames('list-nested-item', {
+    const classNames = [ 'list-nested-item' ];
```

Comment:
We've taken to using `prettier` to format our code so you gotta remove these spaces